docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: docs
+title: Getting Started
+permalink: /documentation/getting-started/
+---
+# Getting Started
+
+This guide will provide you with step-by-step instructions on how to create a functional web test with TestCafe.
+You will learn how to install TestCafe, write a test, run the test on multiple browsers and view test results.
+During creating a test, you will learn the basics of TestCafe API, advancing step by step, from the simple to more elaborate testing techniques.
+
+All tests in this guide target a sample page available at [http://testcafe.devexpress.com/Example](http://testcafe.devexpress.com/Example). You can try running each of them against this page by yourself.
+
+This topic contains the following sections.
+
+* [Setting up Test Project](#setting-up-test-project)
+* [Installing TestCafe](#installing-testcafe)
+* [Writing Test](#writing-test)
+  * [General Code Structure](#general-code-structure)
+  * [Performing Actions on Page](#performing-actions-on-page)
+  * [Assertions](#assertions)
+  * [Dialog Handling](#dialog-handling)
+  * [Authentication](#authentication)
+* [Running Test](#running-test)
+* [Viewing Test Results](#viewing-test-results)
+
+## Setting up Test Project
+
+A TestCafe project is just a file system directory that contains *test fixtures* - JavaScript files with tests.
+It is recommended to create this directory as a subdirectory of the project you are going to test.
+
+This tutorial will demonstrate how to test not a developing project, but the published web page.
+To create tests for this page, create the *my-project* directory anywhere on your computer and the *tests* subdirectory in it.
+You can do this from a command line.
+
+```bash
+mkdir -p my-project\tests
+```
+
+## Installing TestCafe
+
+Make sure that Node.js and npm are installed on your computer. Then go to the tested project directory (*my-project*) and run a single command:
+
+```bash
+npm install --save-dev testcafe
+```
+
+This will create the *node_modules* directory in your current directory (if the first one doesn't exist yet) and will download the TestCafe package to that directory.
+
+## Writing Test
+
+### General Code Structure
+
+TestCafe organizes tests into fixtures. Test fixtures are JavaScript files that contain one or more tests for a single target URL.
+Therefore, you will usually have one test fixture per website page/route.
+
+To create a test fixture, just create a .js file in the *tests* directory and add code to the file.
+
+The following snippet illustrates basic elements of a TestCafe fixture:
+
+* fixture declaration - is added by using the the `fixture` keyword;
```

Comment:
`fixture` is not a keyword. It's a function, which also can be optionally used as a ES6 template literal tag (whichever you prefer more)